static/progressive-web-app.js

```diff
@@ -43,29 +42,21 @@ self.addEventListener("activate", (event) => {
   self.clients.claim(); // Take control of all pages immediately
 });
 
-// Fetch event: Respond from cache or network
+// Fetch event (try network first, if offline return from cache)
 self.addEventListener("fetch", (event) => {
-  const url = new URL(event.request.url);
-
-  // If the request has query parameters, bypass the cache
-  if (url.search) { 
-    event.respondWith(fetch(event.request));
-    return;
-  }
-
   event.respondWith(
-    caches.match(event.request).then((cachedResponse) => {
-      if (cachedResponse) {
-        return cachedResponse;
-      }
-      return fetch(event.request)
-        .then((networkResponse) => {
-          return networkResponse;
-        })
-        .catch((error) => {
-          console.error("[Service Worker] Network request failed:", error);
-          return null;
+    fetch(event.request)
+      .then((networkResponse) => {
+        return networkResponse;
+      })
+      .catch(() => {
+        return caches.match(event.request).then((cachedResponse) => {
+          if (cachedResponse) {
+            return cachedResponse;
+          } else {
+            return null;
+          }
         });
-    })
+      })
```

Comment:
Looks like you need to run formatting

Reply:
> Looks like you need to run formatting

This is formatted, the weird indentation is because it's a `.catch` statement if you look closer.